docs/reference/field-types/array.md

```diff
@@ -0,0 +1,81 @@
+
+This is useful for collections of structured data that clearly belong to a parent document but won't have relationships to other documents, such as multiple sets of contact information for business locations or tabs in a widget.
+
+See the [relationship](relationship.md) field if you exclusively, or primarily, need to indentify a series of other pieces or pages.
+
+## Module field definition
+
+All fields in a piece or page module use their object key as their database field name (e.g., `contactInfo` below).
+
+```javascript
+// In a module's `field.add` configuration:
+contactInfo: {
+  label: 'Contact information',
+  type: 'array',
+  titleField: 'city',
+  fields: {
+    add: {
+      city: {
+        type: 'string',
+        label: 'City'
+      },
+      email: {
+        type: 'email',
+        label: 'Email address'
+      }
+    }
+  }
+}
+```
+
+## Settings
+
+### Required
+
+|  Property | Type | Default | Description |
+|-----------|-----------|-----------|-----------|
+|`label` | String | n/a | Sets the visible label for the field in the UI. |
+|`type` | String | n/a | Specifies the field type (`array` for this type) |
+|`fields` | Object | n/a | The field schema for the array items. See below. |
+
+### Optional
+
+|  Property | Type   | Default | Description |
+|-----------|-----------|-----------|-----------|
+|`help` | String | n/a | Help text for the content editor |
+|`htmlHelp` | String | n/a | Help text with support for HTML markup | universal |
+|`min` | Integer |  n/a | The minimum number of entries required in the array |
+|`max` | Integer |  n/a | The maximum number of entries allowed in the array |
+|`required` | Boolean | `false` | If `true`, the field is mandatory |
+|`titleField` | String |  n/a | The name of one of the array schema fields. If provided, the user interface will use the value of that field as a label for the array tabs. |
+
+::: tip NOTE
+- If there is no `titleField` value, the items are numbered.
+- Setting `titleField` is recommended to improve clarity for content editors.
+- `titleField` can access `relationship` sub-field documents by using dot notation (e.g., `_team[0].mascot`).
+:::
+
+<!-- TODO: The following settings are likely to return, but are not yet implemented. -->
+<!-- |contextual | Boolean | `false` | If `true`, it will prevent the field from appearing in the editor modal | -->
+<!-- |readOnly | Boolean | `false` | If `true`, prevents the user from editing the field value | -->
+
+## Configuring the array field schema
+
+Array field schema are generally configured the same way as the module's overall field schema is configured. The module's schema is configured in the `fields` section in the `add` object. Similarly, Array field schema are configured in `fields` section, using the `add` object to add the actual fields. Both use the field names as keys in the `add` object. Both can contain all field types, including nested `array` fields.
```

Comment:
Array field schemas, not schema.

When discussing array fields, the `fields` subproperty should not be referred to as a "section" because we're not talking about a module and its peers are not "sections."

Reply:
> When discussing array fields, the fields subproperty should not be referred to as a "section" because we're not talking about a module and its peers are not "sections."

This might be a thing to establish a consistent vocabulary for.

Reply:
Still would like to see the word "section" not used here since it's not a module section.